src/phase0/state_transition/mod.rs

```diff
@@ -50,9 +50,13 @@ pub enum Error {
     #[error("invalid state root")]
     InvalidStateRoot,
     #[error(
-        "an invalid epoch is not matching to previous epoch {previous} and current epoch {current}"
+        "an invalid epoch {requested} is not matching to previous epoch {previous} and current epoch {current}"
```

Comment:
i'd suggest updating the language a bit:
```suggestion
        "the requested epoch {requested} is not in the required current epoch {current} or previous epoch {previous}"
```

Reply:
ok